lib/lotus/repository.rb

```diff
@@ -671,12 +672,30 @@ def _persisted?(entity)
         !!entity.id
       end
 
-      def _update_timestamps(entity)
+      # Add time create an entity
+      def _update_created_at(entity)
         if entity.class.attributes.include?(:created_at)
           entity.created_at ||= Time.now.utc
         end
       end
 
+      # Add time update an entity
+      def _update_updated_at(entity)
+        if entity.class.attributes.include?(:updated_at)
+          if entity.updated_at
```

Comment:
@mymai91 This can be DRY `entity.updated_at = entity.updated_at || Time.now.utc`